src/platforms/platform.dom.js

```diff
@@ -268,12 +268,21 @@ function unwatchForRender(node) {
 }
 
 function addResizeListener(node, listener, chart) {
-	var expando = node[EXPANDO_KEY] || (node[EXPANDO_KEY] = {});
+	var expando = node[EXPANDO_KEY] = node[EXPANDO_KEY] || {};
 
 	// Let's keep track of this added resizer and thus avoid DOM query when removing it.
 	var resizer = expando.resizer = createResizer(throttled(function() {
 		if (expando.resizer) {
-			return listener(createEvent('resize', chart));
+			var aspectRatio = chart.options.maintainAspectRatio && chart.aspectRatio || null;
```

Comment:
`var aspectRatio = chart.options.maintainAspectRatio;` should be enough

Reply:
To be consistent, I copied this from `resize`: https://github.com/chartjs/Chart.js/blob/e6a0c8690981de60d789f74e605f6d6b4557e3f6/src/core/core.controller.js#L190